apps/site/src/app/(main)/portal/@applicant/SignWaiver.tsx

```diff
@@ -7,7 +7,8 @@ interface SignWaiverProps {
 }
 
 function SignWaiver({ status }: SignWaiverProps) {
-	return status === PortalStatus.waived ? null : (
+	return status === PortalStatus.waived ||
+		status === PortalStatus.confirmed ? null : (
```

Comment:
Chore: we already conditionally render this `SignWaiver` component from `ApplicantPortal`, so could we fix the condition there and remove the condition here?